packages/video_player/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.10.1+7
+
+* Implemented playback speed feature.
+* Bump the minimum Flutter version to 1.2.0.
+* Add template type parameter to `invokeMethod` calls.
+=======
```

Comment:
@JONA-Cureambit can you remove this line? Looks like one of the merges went wrong

Reply:
@recastrodiaz done 👍